src/components/movies-list/movies-list.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { useState } from "react"
+
+import "./movies-list.css";
+import { MovieData } from "../../App"
+import { SelectGenre } from "../select-genre/select-genre";
+import { MovieItem } from "../movie-item/movie-item";
+
+
+export function MoviesList(props: any): React.ReactElement {
+    const [movies, setMovies] = useState([...props.movies]);
+
+    const sortOptions = ['Title', 'Release Date']
```

Comment:
possible we can provide strategy related to sort? `sortOptions = [{ sortName: "name", strategy: callbackFn }]` and use it in movieSorting